aten/src/ATen/native/quantized/cuda/make_per_channel_quantized_tensor.cu

```diff
@@ -0,0 +1,33 @@
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cuda/Loops.cuh>
+
+namespace at {
+namespace native {
+
+Tensor make_per_channel_quantized_tensor_cuda(
+  const Tensor& self,
+  const Tensor& scales,
+  const Tensor& zero_points,
+  int64_t axis) {
+    Tensor dst = at::_empty_per_channel_affine_quantized(
```

Comment:
Can we have a single _empty function that can internally call either per_channel or per_tensor affine? If we can do that, we can remove this function.

Reply:
we do have empty_quantized which can handle both. But the per_(tensor/channel) make functions have different input types which is the main issue. per_channel takes tensors, per_tensor takes double and int.

Also, we already have:
make_per_tensor_quantized_tensor_cpu
make_per_tensor_quantized_tensor_cuda
make_per_channel_quantized_tensor_cpu

so isn't adding make_per_channel_quantized_tensor_cuda the correct move to preserve the structure?